requirements.txt

```diff
@@ -0,0 +1,12 @@
+aiohttp==2.3.3
```

Comment:
Most of these are just  dependencies of torrent-client. Please add them there.

Reply:
Hey @mwatts15 , please could you elaborate as to where i should add them? How are dependencies dealt with in POW as i saw in the Install.md there is no 'requirements.txt'. Thanks!